api/channel.go

```diff
@@ -482,8 +482,8 @@ type ActiveThread struct {
 
 // ActiveThreads returns all the active threads in the guild, including public
 // and private threads.
-func (c *Client) ActiveThreads(guildID discord.GuildID) ([]ActiveThread, error) {
-	var t []ActiveThread
+func (c *Client) ActiveThreads(guildID discord.GuildID) (ActiveThread, error) {
```

Comment:
It's probably best that `type ActiveThread` be named `ActiveThreads` instead,
and the method should return `(*ActiveThreads, error)` instead.

Reply:
Updated